src/conveyor/conveyor.py

```diff
@@ -5,7 +5,7 @@
 from pathlib import Path
 from typing import Sequence
 
-from src.blueprints.datafile import BaseDatafile
+from src.blueprints.datafile import RawDatafile
```

Comment:
Wondering why we've changed from `BaseDataFile` to `RawDataFile` here?

Reply:
I added a step to check and just convey datafiles those newly ingested, and there will be an error indication from mypy if I use BaseDataFile instead of RawDatafile. 
Might need to discuss this with @uoa-noel as Noel mentioned the conveyor could check existing files but it didn't from my test - that's why I added this step.

Reply:
Have talked with Noel about this and I need to test in another branch.